### PR TITLE
model, distsql: add unit tests (exclude distsql/xeval)

### PR DESCRIPTION
**Pull request by jackysp**
Add unit tests for model and distsql (exclude distsql/xeval). 
PTAL @XuHuaiyu

### CODE REVIEW

File: model/model_test.go [hunk 1]
```diff
@@ -79,9 +87,41 @@ func (*testSuite) TestClone(c *C) {
...
+	c.Assert(pkName, Equals, NewCIStr("c"))
+	newColumn := table.GetPkColInfo()
+	c.Assert(newColumn, DeepEquals, column)
+	inIdx := table.ColumnIsInIndex(column)
+	c.Assert(inIdx, Equals, true)
+	tp := IndexTypeBtree
+	c.Assert(tp.String(), Equals, "BTREE")
+	tp = IndexTypeHash
+	c.Assert(tp.String(), Equals, "HASH")
+	tp = 1E5
```
Comment (shenli) [inline model/model_test.go]: What's 1E5?

File: distsql/distsql_test.go [hunk 2]
```diff
@@ -33,25 +33,100 @@ func TestT(t *testing.T) {
 	TestingT(t)
 }
 
-var _ = Suite(&testTableCodecSuite{})
+var _ = Suite(&testDistsqlSuite{})
 
-type testTableCodecSuite struct{}
+type testDistsqlSuite struct{}
 
 // TODO: add more tests.
```
Comment (shenli) [inline distsql/distsql_test.go]: Could we remove the TODO?

File: model/model_test.go (hunk 1 above)
Comment (jackysp) [inline model/model_test.go]: It's just an invalid index type.

File: distsql/distsql_test.go (hunk 2 above)
Comment (jackysp) [inline distsql/distsql_test.go]: Yes, I'll remove this line.

**Comment by shenli** [comment]
LGTM
@tiancaiamao PTAL

**Comment by shenli** [comment]
@tiancaiamao PTAL

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**